packages/server/lib/ipc/ipc.js

```diff
@@ -1,9 +1,13 @@
-const {
-  contextBridge,
-  ipcRenderer,
-} = require('electron')
+const debug = require('debug')('cypress:server:ipc')
 
-contextBridge.exposeInMainWorld('ipc', {
-  on: (...args) => ipcRenderer.on(...args),
-  send: (...args) => ipcRenderer.send(...args),
-})
+if (require('../util/electron-app').isRunningAsElectronProcess({ debug })) {
```

Comment:
Instead of the inline `require` calls, can you do something like this at the top level?

```js
const { isRunningAsElectronProcess } = require('../util/electron-app')
```

Reply:
I think it may be this way to defer compiling the JS, for when the runner relaunches itself from Node -> Electron

Reply:
Ok, then!